test/evaluation/evaluationClient.spec.ts

```diff
@@ -3,211 +3,291 @@ import fs from 'fs';
 import { Optionality, PresentationDefinition } from '@sphereon/pe-models';
 
 import { Status } from '../../lib';
-import { EvaluationClient } from '../../lib/evaluation/evaluationClient';
+import { EvaluationClient } from "../../lib/evaluation/evaluationClient";
```

Comment:
please reduce the change noise of the commit for example if your changing these kinds of things (double-quote with single-quote) avoid it

Reply:
actually, we need a configuration that is shared across all our IDEs for these kinds of things

Reply:
It's supposed to be fixed by eslint. Maybe I forgot to run it.

Reply:
> actually, we need a configuration that is shared across all our IDEs for these kinds of things

@sksadjad @maikel-maas @hrehman-sphereon See https://sphereon.atlassian.net/wiki/spaces/DEV/pages/1570046076/Sphereon+Code+Style ;)
We do have a code style (Google code)

Don't use the settings repo option though !